mpv.py

```diff
@@ -1040,6 +1054,20 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def command_async(self, name, *args, callback=None):
+        """
+        Same as mpv_command, but run the command asynchronously.
+        Once the command ran, the callback will be invoked, if provided.
+        The first argument of the callback will be a boolean value.
+        It will be set to True, if there was an error, False else.
+        The second argument of the callback depends on the command.
+        """
+        args = [name.encode('utf-8')] + [(arg if type(arg) is bytes else str(arg).encode('utf-8'))
+                                         for arg in args if arg is not None] + [None]
```

Comment:
Since this is the second copy of this string encoding mojibake in this file, maybe we should factor out this and the occurrence in MPV.command() above into a non-public method?

Reply:
:+1: Solved in latest push.